code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -396,6 +408,132 @@
 		span_warning("You repair \the [src]."))
 		chair_state = DROPSHIP_CHAIR_FOLDED
 
+/obj/structure/bed/chair/dropship/doublewide
+	name = "doublewide seat"
+	icon_state = "doublewide_chair" //only facing south cause the rest are ugly
+	max_integrity = 130
+	/// Handles the color of the chair 
+	var/chair_color = NO_CHAIR_COLOR
+	/// If the chair can only be sat in by a leader or not
+	var/leader_chair = FALSE
+	/// pixel x shift to give to the buckled mob
+	var/buckling_x = 0
+
+/obj/structure/bed/chair/dropship/doublewide/attack_alien(mob/living/carbon/xenomorph/X, damage_amount = X.xeno_caste.melee_damage, damage_type = BRUTE, damage_flag = "", effects = TRUE, armor_penetration = 0, isrightclick = FALSE)
+	if(X.status_flags & INCORPOREAL)
+		return FALSE
+	if(LAZYLEN(buckled_mobs))
+		unbuckle_all_mobs(TRUE)
+	return ..()
+
+/obj/structure/bed/chair/dropship/doublewide/welder_act(mob/living/user, obj/item/I)
+	welder_repair_act(user, I, 130, 1 SECONDS, 0, SKILL_ENGINEER_METAL, 1)
+	chair_state = DROPSHIP_CHAIR_FOLDED
+	update_icon()
+	return
+
+/obj/structure/bed/chair/dropship/doublewide/update_icon_state()
+	. = ..()
+	var/percentage = (obj_integrity / max_integrity) * 100
```

Comment:
Why is this in update_icon_state

Reply:
icon state is handled by health, not sure where else to put this, don't really remember the precedent